pkg/graveler/ref/manager.go

```diff
@@ -13,26 +15,36 @@ import (
 // IteratorPrefetchSize is the amount of records to maybeFetch from PG
 const IteratorPrefetchSize = 1000
 
+const MaxBatchDelay = time.Millisecond * 3
```

Comment:
How did we reach this number?
Maybe just add a comment describing it's a guesstimate.

Reply:
I'd prefer it to be a configurable parameter, if we believe it can be tuned to affect performance.